src/app/multi-editor/shared/services/multi-api.service.ts

```diff
@@ -50,24 +50,32 @@ export class MultiApiService extends CommonApiService {
       ).map(res => res.json());
   }
 
-  previewActions(userActions: UserActions, pageNumber: number, size: number): Observable<RecordsPreview> {
+  previewActions(userActions: UserActions, page: number, size: number): Observable<RecordsPreview> {
```

Comment:
maybe new name since it's calling `/search` now.

Reply:
Create `PreviewActionsOptions` kinda like interface (where `page` is optional) and use it as type of single parameter of this function. In that way you don't need `if` here.